### DISCUSSION TITLE
configuring the router for IPV6 address for AdGuard Home

### DISCUSSION THREAD

**Discussion by hutchwilco** [AdguardTeam/AdGuardHome#2885] [Q&A]
I have a Fritzbox router which alongside IPv4 DNS address input, allows for an IPv6 DNS address to be specified.

Is this necessary, possible, or even a thing?

My limited experience says my local network doesn't use IPv6, but since its an option... is that option there for the case where my chosen DNS server was external (my AdGuard Home instance is running in Home Assistant, on my local network)

Thanks

**Comment by ameshkov** [comment]
> Is this necessary, possible, or even a thing?

It is definitely not necessary if your local network has IPv4, you should be good with just IPv4 and empty IPv6

**Comment by triksmelb** [comment]
so to clarify, there is no way to get local IPv6 address out of AdGuard to use in my router (SonicWall) which has provision for IPv4 and IPv6 in DNS server settings?
<img width="859" alt="Screen Shot 2022-05-26 at 12 59 51 pm" src="https://user-images.githubusercontent.com/12489962/170407250-d628e458-51bb-4269-900e-0bd5d3240d5b.png">.
